VoidCases/turkish.json

```
@@ -1,119 +1,264 @@
+    "settings_language": "Dil",
+    "trading_requestexpired": "Ticaret isteği süresi doldu",
+    "is_permanent": "KALICI MI?",
+    "no_items": "Hiçbir öğeniz yok!",
+    "trading_mismatch": "Öğe/para eşleşmemesi",
+    "settings_3d2dinfo_desc": "Infobox HUD'a yazılsın mı?",
+    "trading_request": "Ticaret isteği",
+    "trading_success": "%s ile başarılı bir şekilde ticaret yapıldı",
+    "just_listed": "Satış için yeni listelediniz:",
+    "trading_inprogress": "Aktif bir ticaret isteğiniz var!",
+    "drop_chance": "DÜŞÜŞ ŞANSI",
+    "scifi_crate": "Bilim Kurgu Sandığı",
+    "trading_playerdisconnect": "Oyuncu bağlantısı kesildi",
+    "featured_items": "ÖNE ÇIKAN ÖĞELER",
+    "annouce": "Duyuru",
+    "item_details": "ÖĞE DETAYLARI",
+    "trading_confirmagain": "Tekrar onayla",
+    "trading_itemchanged": "Öğeler değiştirildi",
+    "trading_send": "GÖNDER",
+    "pointshopsh_item": "Pointshop SH Öğesi",
+    "not_enough_money": "Bunu karşılayacak kadar paranız yok!",
+    "settings_language_desc": "Eklentinin kullanacağı dil",
+    "key_skin": "ANAHTAR SKINI",
+    "mp_not_enough_items": "Satıcı, belirtilen miktarda satış için yeterli öğeye sahip değil!",
+    "plastic_crate": "Plastik Kasa",
+    "price_per_item": "BİRİM FİYAT",
+    "wooden_crate": "Tahta Kasa",
+    "pointshop_item": "Pointshop 1 Öğesi",
+    "winnable_items": "KAZANILABİLİR ÖĞELER",
+    "entity_class": "VARLIK SINIFI",
+    "month": "Ay",
+    "day": "Gün",
+    "money": "Para",
+    "world_opening": "DÜNYADA AÇILMA",
+    "trading_waiting": "BEKLİYOR",
+    "usergroups": "İZİN VERİLEN KULLANICI GRUPLARI",
+    "couldnt_purchase": "Satın alınamadı:",
+    "weapon": "Silah",
+    "trading_playerend": "%s takası terk etti",
+    "delete": "SİL",
+    "create_market": "ÖĞE SAT",
+    "no": "Hayır",
+    "trading_toply": "hedeflenen oyuncuya: %s",
+    "create_new": "YENİ OLUŞTUR...",
+    "general": "Genel",
+    "model_path": "MODEL YOLU",
+    "key": "Anahtar",
+    "item_fov": "MODEL GÖRÜŞ ALANI",
+    "editing": "DÜZENLENİYOR: %s",
+    "bulk_purchase": "TOPLAM ALINACAK ÖĞELER",
+    "custom_image_id": "ÖZEL RESİM KİMLİĞİ",
+    "scifi_key": "Bilim Kurgu Anahtarı",
+    "save": "KAYDET",
+    "weapon_skin": "Silah Kaplaması",
+    "delete_confirm_title": "%s'ı sil",
+    "cancel": "İPTAL",
+    "day_short": "Gün",
+    "item_icon": "ÜRÜN İKON TÜRÜ",
+    "item_preview": "ÜRÜN ÖNİZLEMESİ",
+    "item_limitations": "SINIRLAMALAR",
+    "case_place": "Kutu yerleştir",
+    "weapon_class": "SİLAH SINIFI",
+    "just_bought": "Şimdi aldınız:",
+    "just_got_bought": "Ürününüz şimdi satıldı:",
+    "trading_end": "Ticaret sona erdi",
+    "hour_short": "Saat",
+    "limit_reached_perma": "Sadece %s kalıcı silah donatabilirsiniz!",
+    "trading_requestsent": "Ticaret isteği gönderildi",
+    "dont_annouce": "Duyurulmasın",
+    "annouce_win": "%s sadece %s kutusundan çıkardı!",
+    "opening_case": "KUTU AÇILIYOR: %s",
+
+    "edit_items": "Öğeleri Düzenle"
```

Missing comma here

---

Sir, seems same with english.json line 182 space.

---

The space is fine, but you are missing a comma, should be like this:
```suggestion
    "edit_items": "Öğeleri Düzenle",
```